reactive/kotlinx-coroutines-rx2/api/kotlinx-coroutines-rx2.api

```diff
@@ -67,7 +67,9 @@ public final class kotlinx/coroutines/rx2/RxObservableKt {
 }
 
 public final class kotlinx/coroutines/rx2/RxSchedulerKt {
-	public static final fun asCoroutineDispatcher (Lio/reactivex/Scheduler;)Lkotlinx/coroutines/rx2/SchedulerCoroutineDispatcher;
+	public static final fun asCoroutineDispatcher (Lio/reactivex/Scheduler;)Lkotlinx/coroutines/CoroutineDispatcher;
+	public static final synthetic fun asCoroutineDispatcher (Lio/reactivex/Scheduler;)Lkotlinx/coroutines/rx2/SchedulerCoroutineDispatcher;
```

Comment:
@elizarov is this "synthetic" ok?

Reply:
Yes. That is because it is HIDDEN. Old code that was compiled against the previous version of the library will still work.